provision/golang.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-sudo mkdir /go/ &&\
-export GOPATH=/go/ &&\
+source "./env.bash"
+
+sudo -E mkdir "${GOPATH}" && \
 go get -u github.com/jteeuwen/go-bindata/... && \
 go get -u github.com/google/gops && \
-go get -u github.com/golang/protobuf/protoc-gen-go && \
-sudo ln -sf /go/bin/* /usr/local/bin/
+go get -u github.com/golang/protobuf/protoc-gen-go
```

Comment:
Why do you delete the `ln` command here?

Reply:
It's no longer needed because `GOPATH/bin` belongs to PATH as well